### PR TITLE
Fixes broken link

### LINKED ISSUE #60: cheat sheet link need to update
## Bug Report
*Problem*
[This page ](https://github.com/kubernetes-sigs/slack-infra/blob/main/CONTRIBUTING.md) has one broken link of `contributor cheat sheet` which needs to fix.
 

*solution*
need to locate the correct path of README.

/assign 
/kind bug
/kind documentation

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>
Fixes #60

### CODE REVIEW

**Review by cpanato — COMMENTED** [review]
/ok-to-test

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
 - [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
-- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet.md) - Common resources for existing developers
+- [Contributor Cheat Sheet](https://github.com/kubernetes/community/blob/master/contributors/guide/contributor-cheatsheet/README.md) - Common resources for existing developers
```
Comment (cpanato) [inline CONTRIBUTING.md]:
```suggestion
- [Contributor Cheat Sheet](https://www.kubernetes.dev/docs/contributor-cheatsheet/) - Common resources for existing developers
```

**Review by cpanato — CHANGES_REQUESTED** [review]
thanks

a small request

File: CONTRIBUTING.md (hunk 1 above)
Comment (amolmote) [reply CONTRIBUTING.md]: ok I commit your suggestion and squash it. thank you @cpanato

**Review by cpanato — APPROVED** [review]
/lgtm
/approve

**Comment by cpanato** [comment]
/assign @nikhita

for approval :)

**Review by cpanato — COMMENTED** [review]
/lgtm

**Comment by amolmote** [comment]
@cpanato can we use this https://k8s.dev/cheatsheet shorter version of link here. Bobby suggested me this link.
what do you think?

**Comment by cpanato** [comment]
Sound great

**Comment by amolmote** [comment]
> Sound great
done @cpanato, I have updated the link with shorter format. it looks good

**Review by cpanato — APPROVED** [review]
/lgtm

**Comment by cpanato** [comment]
lets get this merged 
/assign @nikhita

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

/woof

**Pull request merged**